pandas/core/_algos/needsname.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
```

Comment:
i would put this in pandas.core.algos.methods ?

pandas.core.algos.ops

Reply:
changed to core._algos.transforms, for shape-preserving functions